README.md

```diff
@@ -1,27 +1,66 @@
 # ForemanPuppet
 
-Adds Puppet External node classification funcionality for [Foreman](theforeman.org).
+[![Gem Version](https://img.shields.io/gem/v/foreman_puppet.svg)](https://rubygems.org/gems/foreman_puppet)
+[![Code Climate](https://codeclimate.com/github/theforeman/foreman_puppet/badges/gpa.svg)](https://codeclimate.com/github/theforeman/foreman_puppet)
+[![GPL License](https://img.shields.io/github/license/theforeman/foreman_puppet.svg)](https://github.com/theforeman/foreman_puppet/blob/master/LICENSE)
 
+This plugin adds Puppet External node classification functionality to [Foreman](theforeman.org).
+
+* Website: [theforeman.org](http://theforeman.org)
+* Support: [Foreman support](http://theforeman.org/support.html)
+
+## Features
+
+* Puppet dashboards
+* Puppet
+    * Environments
+    * Classes
+    * Config groups
+    * Smart Class parameters
+* Host / Hostgroup
+    * Classes assignment
+    * Environment assignment
+    * Puppet proxy assignment
+* Puppet ENC (external node classifier)
+* Smartproxy Puppet status/tab
+* Template hostgroup / environment
+
+## Compatibility
+
+Foreman 3.0 will be the first release where the Puppet functionality is fully extracted.
```

Comment:
Puppet CA and Reports are still a core feature. Is `fully extracted` misleading here?

Reply:
Hmm, good point. How about: 
```suggestion
Foreman 3.0 will be the first release where the Puppet functionality that is included in this plugin will be unavailable in Foreman core.
```

Reply:
I've just used `mostly removed` and added the remaining features in core in a the section above.

Reply:
It is, but there is a plan to extract it here before 3.0 :)